### ISSUE TITLE
xmrig.exe flaged by windows defender

### ISSUE THREAD

**Issue by sparta3407** [xmrig/xmrig#124]
any chance to encrypt xmrig.exe to stop windows defender detection !

**Comment by robme** [comment]
Windows Defender is very aggressive about this.

I kept trying to add an exception, but it was impossible, because to add an exception I needed the file to add. So I added the zip as an exception, but when I extracted it, xmrig.exe was always removed immediately.

So I disabled Windows Defender and extracted the zip, and then tried to add xmrig.exe as an exception, and it **still** wouldn't let me - gave me [this message](https://i.imgur.com/QzHvRiF.png).

However, Windows Defender still flagged it when it was disabled, but because it wasn't automatically quarantined I had the option to "Allow on device". Windows Defender still considers it a risk and keeps complaining about it, but at least I can run the miner now.

I understand why they flagged it, because when it's put on someone's computer without their knowledge then it will make their computer very slow, but it's insane how difficult it is for someone that intentionally downloads it.

Now if I actually have a threat, I will be less likely to notice because of virus scanner who cried wolf. I permanently have a big red "threats found" all the time.
